localstack/services/stepfunctions/stepfunctions_starter.py

```diff
@@ -16,6 +16,8 @@
 # todo: will be replaced with plugin mechanism
 PROCESS_THREAD = None
 
+default_region = "us-east-1"
```

Comment:
Let's better remove this and use `stepfunctions_listener.default_region` in line 30 below..